### PR TITLE
docs: Fix TOC navigation links

### PR DESCRIPTION
**Pull request by dhonx**

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -18,15 +18,15 @@ you can do in V.
...
 <table>
 	<tr>
-		<td><a href='#hello-world'>1. Hello world</a></td>
-		<td><a href='#comments'>2. Comments</a></td>
-		<td><a href='#functions'>3. Functions</a></td>
-		<td><a href='#constants--variables'>4. Variables</a></td>
-		<td><a href='#primitive-types'>5. Primitive types</a></td>
-		<td><a href='#strings'>6. Strings</a></td>
-		</tr>
+		<td><a href='#hello-world'>Hello world</a></td>
```
Comment (dhonx) [inline doc/docs.md:21]: Also... I don't think we should include a number

Comment (medvednikov) [reply doc/docs.md:21]: They are needed so that it's clear it goes from left to right

Comment (dhonx) [reply doc/docs.md:21]: Why don't use lists instead?

Comment (medvednikov) [reply doc/docs.md:21]: The don't work inside tables, the layout is off

**Pull request merged**